apps/mobile/src/app/(home)/settings/networks/index.tsx

```diff
@@ -39,14 +44,16 @@ export default function SettingsNetworksScreen() {
   return (
     <SettingsScreenLayout>
       {defaultNetworkPreferences.map(network => (
-        <NetworkCell
-          key={network}
-          caption={settings.networkPreference.id === network ? t`Enabled` : t`Disabled`}
-          icon={getNetworkIcon(network)}
-          isSelected={settings.networkPreference.id === network}
-          onChangeNetwork={() => onChangeNetwork(network)}
+        <Action.Cell
```

Comment:
I think that these types of maps like `defaultNetworkPreferences.map` seem more efficient but actually make things harder to follow and add complexity. 

In this case if we want to minimise repetition we could have a `NetworkCell` that wraps `Action.Cell` and just call it more times: 

```
    <SettingsScreenLayout>
        <NetworkCell network='mainnet' icon={<GlobeIcon />} />
        <NetworkCell network='testnet' icon={<TestTubeIcon />} />
        <NetworkCell network='signet'  icon={<PlaygroundFormsIcon />}/>
   </SettingsScreenLayout>

... 
// change defaultNetworkPreferences to a type 
type Networks = 'mainnet' | 'testnet' | 'signet';

interface NetworkCell {
 network: Networks;
icon: React.ReactNode;
}

function NetworkCell({network, icon}: NetworkCell){
   return  (<Action.Cell> 
          icon={icon}
          title={i18n._(capitalize(network))}
          caption={settings.networkPreference.id === network ? t`Enabled` : t`Disabled`}
          onPress={() => onChangeNetwork(network)}>
           <Action.Radio isSelected={settings.networkPreference.id === network} />
       
   </Action.Cell>)
}
```